CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 
 - feat: Dynamic endpoints and token to use official Tesla API (self-hosted or from third party provider) (#3903 - @jlestel)
 
+### New MQTT Topics
```

Comment:
we do not list these in changelog

Reply:
Ah okay, I saw it in the old [changelog for `trunk_open` and `frunk_open`](https://github.com/teslamate-org/teslamate/blob/master/CHANGELOG.md#1190---2020-04-19), but maybe we no longer do it; I'll remove it.